src/ray/core_worker/core_worker.h

```diff
@@ -1955,6 +1960,9 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   /// Maps serialized runtime env info to **immutable** deserialized protobuf.
   mutable utils::container::ThreadSafeSharedLruCache<std::string, rpc::RuntimeEnvInfo>
       runtime_env_json_serialization_cache_;
+
+  // Reconstructable actors owned by this worker.
```

Comment:
```suggestion
  // Restartable actors owned by this worker.
```

also indicate the lifecycle of entries in the map